.github/workflows/release.yml

```
@@ -35,10 +33,14 @@ jobs:
           token: ${{ secrets.GITHUB_TOKEN }}
 
   deploy_production:
+    name: Deploy Web (Production)
     runs-on: ubuntu-latest
     needs: create_release
     env:
       VERSION_TAG: ${{ needs.generate_tag.outputs.version }}
```

actionlint (1.7.10)

[error] property "generate_tag" is not defined in object type {create_release: {outputs: {}; result: string}}

(expression)